content/10-minute-tutorial.md

```diff
@@ -1,3 +1,324 @@
+
+1. Right click on the Test folder, select **New > Directory** and name this directory `resources`.
+
+2. Right click the folder and select **Mark directory as > Test Resources Root**.
+
+3. You can add subdirectories as needed. Create a subdirectory for your project in `src/test/resources/`
+
+Our project structure is now as follows:
+
+```
+├── pom.xml             (containing our Cucumber and JUnit dependencies)
+└── src
+    ├── main
+    │   └── java        (marked as sources root)
+    │   └── resources   (marked as resources root)
+    └── test
+        ├── java        (marked as test sources root)
+        └── resources   (marked as test resources root)
+                └── <project>
+```
+
+## Creating a Feature
+
+To create a feature file:
+
+1. Open the project in your IDE (if needed) and right-click on the `src/test/resources/<project>` folder.
+
+2. Select **New > File**
+
+3. Enter a name for your feature file, and use the `.feature` extension. For instance, `belly.feature`.
+
+Files in this folder with an extension of `.feature` are automatically recognised as feature files. Each feature file describes a single feature, or part of a feature.
+
+Open the file (double click it) and add the feature description, starting with the `Feature` keyword and an optional description.
+
+For instance:
+```gherkin
+Feature: Belly
+  Optional description of the feature
+```
+
+## Creating a Scenario
+
+Scenarios are added to the feature file, to define examples of the expected behaviour. These scenarios can be used to test the feature.
+Start a scenario with the `Scenario` keyword and add a brief description of the scenario. To define the scenario, you have to define all of its steps.
+
+## Defining Steps
+
+These all have a keyword (`Given`, `When`, and `Then`) followed by a step. The step is then matched to a [step definition](/step-definitions/) which map the plain text step to programming code.
+
+The plain text steps are defined in the [Gherkin](/gherkin/) language. Gherkin allows developers and business stakeholders to describe and share the expected behaviour of the application. It should not describe the implementation.
+
+The feature file contains the Gherkin source.
+
+- The `Given` keyword precedes text defining the context; the known state of the system (or precondition).
+
+- The `When` keyword precedes text defining an action.
+
+- The `Then` keyword precedes text defining the result of the action on the context (or expected result).
+
+For instance:
+```gherkin
+  Scenario: a few cukes
+    Given I have 42 cukes in my belly
+    When I wait 1 hour
+    Then my belly should growl
+```
+
+# Running the test
+
+You can run the test from IntelliJ:
+
+1. Right click the feature file
+
+2. Select **Run 'Feature: tutorial'** from the context menu.
+
+{{% note "Formatter error"%}}
+If you are using Cucumber v2.0.0 or higher, when running the test for the first time you might get an error message that mentions `CucumberJvmSMFormatterUtil`.
```

Comment:
`If you are using Cucumber v2.0.0 or higher` can be removed we are using 2.0.0. You've specified that in the tutorial.

Reply:
Good point. Will update